lib/movable_ink/aws/eks.rb

```diff
@@ -0,0 +1,52 @@
+require 'aws-sdk-eks'
+require 'yaml'
+
+module MovableInk
+  class AWS
+    module EKS
+      def eks(region: my_region)
+        @eks_client ||= {}
+        @eks_client[region] ||= Aws::EKS::Client.new(region: region)
+      end
+
+      def generate_kubeconfig(region: my_region, cluster_name:)
+        client = eks(region: region)
+        resp = client.describe_cluster({ name: cluster_name })
```

Comment:
We should wrap the describe call in `run_with_backoff` and handle the case where we don't get back a cluster.

Reply:
That's a good point, added